states/huntsman/ready.py

```diff
@@ -32,3 +32,7 @@ def on_enter(event_data):
                 pocs.next_state = 'scheduling'
             else:
                 pocs.next_state = 'twilight_flat_fielding'
+
+    # Prepare the cameras if we are about to take some exposures
+    if pocs.next_state != 'parking':
+        pocs.observatory.prepare_cameras()
```

Comment:
```suggestion
    if pocs.next_state != 'parking':
        pocs.say("Those cameras are hot! Let's turn on the cooling. This might take a few minutes.")
        pocs.observatory.prepare_cameras()
```